ote/src/clj/ote/transit_changes/change_history.clj

```diff
@@ -0,0 +1,51 @@
+(ns ote.transit-changes.change-history
+  "Store detected changes once to db."
+  (:require
+    [ote.time :as time]
+    [jeesql.core :refer [defqueries]]
+    [specql.core :as specql]
+    [ote.util.collections :refer [map-by count-matching]]
+    [ote.util.functor :refer [fmap]]
+    [digest]))
+
+(def history-columns #{:gtfs/change-key :gtfs/route-hash-id :gtfs/different-week-date})
+
+(defn create-change-key-from-change-data
```

Comment:
Code formatting: indentation a bit off? :)

Reply:
It is never a good idea to format your hard drive and install everything again.

Reply:
My editor says that alles bra. Need to figure out why it is not messed up in my machine.

Reply:
Fixed now. I had tab tab and changed it to tab -> two space